packages/backend-app-api/src/services/implementations/events/eventServiceFactory.ts

```diff
@@ -14,16 +14,19 @@
  * limitations under the License.
  */
 
-import { EventBroker } from './EventBroker';
+import {
+  coreServices,
+  createServiceFactory,
+} from '@backstage/backend-plugin-api';
+import { DefaultEventBroker } from '@backstage/plugin-events-node';
 
-/**
- * Publishes events to be consumed by subscribers for their topic.
- * The events can come from different (external) sources
- * like emitted themselves, received via HTTP endpoint (i.e. webhook)
- * or from event brokers, queues, etc.
- *
- * @public
- */
-export interface EventPublisher {
-  setEventBroker(eventBroker: EventBroker): Promise<void>;
-}
+/** @public */
+export const eventServiceFactory = createServiceFactory({
```

Comment:
Maybe, it should be called `eventsServiceFactory` instead as the service is called `events`